reactor-core/src/main/java/reactor/core/publisher/Sinks.java

```diff
@@ -710,7 +748,11 @@ public Emission getReason() {
 		 * @deprecated to be removed shortly after 3.4.0-RC1. Use {@link #tryEmitEmpty()} and handle the result.
 		 */
 		@Deprecated
-		void emitEmpty();
+		default void emitEmpty() {
+			emitEmpty(EmitFailureHandler.FAIL_FAST);
+		}
+
+		void emitEmpty(EmitFailureHandler failureHandler);
```

Comment:
missing javadoc. insist on the fact that this should rely on `tryEmitEmpty` (see https://github.com/reactor/reactor-core/pull/2405#discussion_r496589276)